src/generators/index.mjs

```diff
@@ -23,6 +24,7 @@ export const publicGenerators = {
   'api-links': apiLinks,
   'orama-db': oramaDb,
   'llms-txt': llmsTxt,
+  jsx,
```

Comment:
```suggestion
  'jsx': jsx,
```

Reply:
What's the difference?

Reply:
same but rest use so why not use same

Reply:
The rest use different keys than the variable names, so we wouldn't be able to use this syntax. 

I don't think we need to change it.